app/scripts/modules/core/src/managed/ArtifactsList.tsx

```diff
@@ -2,32 +2,31 @@ import React from 'react';
 import classNames from 'classnames';
 
 import { IManagedArtifactSummary, IManagedArtifactVersion } from '../domain/IManagedEntity';
-import { ISelectedArtifact } from './Environments';
+import { ISelectedArtifactVersion } from './Environments';
 import { Pill } from './Pill';
-import { parseName } from './Frigga';
 
 import styles from './ArtifactRow.module.css';
 
 interface IArtifactsListProps {
   artifacts: IManagedArtifactSummary[];
-  artifactSelected: (artifact: ISelectedArtifact) => void;
-  selectedArtifact: ISelectedArtifact;
```

Comment:
with the increasing number of places/contexts we refer to things like 'artifact(s)' generally, it's starting to get very ambiguous. Part of the refactor that happened here was moving to be more precise by referring to a _version_, which is really what we're working with (vs. the overall 'artifact' which encompasses all the versions that exist)